src/io/files.js

```diff
@@ -125,7 +125,7 @@ p5.prototype.loadFont = function(path, onSuccess, onError) {
     }
     /*jshint multistr: true */
     var exp =/\/[a-zA-Z]*((.ttf)|(.otf)|(.woff)|(.woff2))$/i;
-    if(!exp) {
+    if(!exp.test(path)) {
```

Comment:
I don't see any way for the regular expression itself to be falsy, so I'm guessing that the original author meant to test it against `path`, since it's `exec`'d against `path` on line 131?